### PR TITLE
Upgraded and locked Rubocop at 0.60.0.

### PR DESCRIPTION
**Pull request by dblock**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ group :development, :test do
   gem 'rake'
   gem 'rdoc'
   gem 'rspec', '~> 3.0'
-  gem 'rubocop', '~> 0.59', require: false
+  gem 'rubocop', '0.60.0', require: false
```
Comment (LeFnord) [inline Gemfile]: please, don't lock it

**Comment by LeFnord** [comment]
@dblock … please revert the locking, then this branch will be mörged first, then the two others, thanks

**Comment by dblock** [comment]
@LeFnord Thanks for a quick review. 

I think we should lock it. Yesterday's build with 0.59 worked, but today the same code fails to build because it's bringing a newer version. 

Does this ^ change anything? Let me know what you'd like me to do.

**Comment by LeFnord** [comment]
@dblock … yeap agree with your arguments, but as I said, minor rubocop updates are easier to handle then big ones, and IMHO it is a maintainance task 😉 and should be done from time to time

**Comment by dblock** [comment]
Rubocop doesn't quite follow semver, I changed to `~> 0.60.0`, LMK if that's acceptable?

**Comment by dblock** [comment]
Marked HEAD to allow failures. Right now fails without #717

**Comment by LeFnord** [comment]
mmh, please set it to `~> 0.60` … Batzov has its own idea of SemVer 😉 

yeap, that is fine, I'll mörge, so that #717 is a clean feature request, thanks again

**Pull request merged**